src/www/ui_components/server-card.html

```diff
@@ -118,6 +118,13 @@
         color: var(--disabled-text-color);
         background: transparent;
       }
+
+      #unsupportedCipherMessage {
+        float: left;
```

Comment:
This is not RTL-friendly. Do we really need to use `float`?
Maybe change to use [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) on card-actions instead. (and maybe get rid of the `text-align: right` there too)

Reply:
Done.